src/test/java/com/exasol/adapter/document/files/extension/ExtensionIT.java

```diff
@@ -184,4 +190,18 @@ private Optional<String> getInDatabaseS3Address() {
                 .map(endpoint -> exasolTestSetup.makeTcpServiceAccessibleFromDatabase(endpoint))
                 .map(InetSocketAddress::toString);
     }
+
+    // Remove this once a new version is released that supports the new VS name
```

Comment:
Can you specify a little bit more precisely how to identify if the described event has occurred? 
Can we specify a release date or version number of the "new" version? 
Or how to detect if (an arbitrary) new version matches the described criterion?